static/js/zamboni/devhub.js

```diff
@@ -376,6 +380,17 @@ function addonFormSubmit() {
     })(parent_div);
 }
 
+function refreshAddonBasicInformation($form) {
+    // Retrieve the div containing the updated name and then its value
+    update_name_div = $form.find('[data-name=name]')
+    updated_name = update_name_div.text()
+    // Get the element to update in the top-left quadrant and update it with the new name
+    // TODO if possible change the H2 tag to have an id attribute. Need to identify where the H2 tag is generated
+    // For the time being we rely on the structure of the page
+    name_h2_tag = $('[id=l10n-menu]').next()
```

Comment:
Yeah, we definitely want to fix this before merging. The `h2` you're looking for is probably in `src/olympia/devhub/templates/devhub/addons/edit.html` but double check that. I suggest setting a `class="addon-name"` on it which we already use in Personas edit pages.